google/cloud/firestore_v1/vector.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List, Union
+
+
+class Vector:
```

Comment:
Respond in another thread: the order compare information is now depending on the ARRAY type (Implement in TypeOrder) though agreed this sounds like a good thing to do after organizing the type code.

Reply:
implementing the Sequence methods shouldn't rely on ordering. And adding these feels more important if the underlying `._value` array is private (shouldn't users have a way to access the raw values if the need to?)

I agree this can come later though if needed

Reply:
Make sense. Updated